realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -1865,6 +1873,121 @@ public long count() {
         return createRealmResults(query, sortDescriptor, null, false);
     }
 
+    /**
+     * Sort the query result by the specific field name in ascending order. This predicate can be applied multiple
```

Comment:
`This predicate can be applied multiple times, but only the last specified sorting order will be used.`
Would it be better we just throw in this case?
Also, this can be called on different fields, what will be the behavior then? like:
`query.sort("A").sort("B").sort("C");`

Reply:
In each case a new sort descriptor is created replacing the existing ones, meaning that only `sort("C")` is applied. You might be right that throwing might be a safer route, then we can re-evaluate if people really want to override older sortings (I suspect that no-one will run into it).